### DISCUSSION TITLE
Customize Remix Server log

### DISCUSSION THREAD

**Discussion by thomasverleye** [remix-run/remix#3858] [Q&A] [answered]
Is there an option to customize the log output of Remix?

Currently it looks something like this:
```
GET / 200 - - 18.098 ms
GET / 200 - - 5.286 ms
GET / 200 - - 2.659 ms
GET / 200 - - 3.095 ms
GET / 200 - - 25.398 ms
```

We would like to have a timestamp added here so it would be a little easier to debug things when testing things out on staging servers.

**Comment by kiliman** [comment, marked as answer]
Looks like you're using Remix App Server. This is an Express server, but not open to customization. It's using `morgan` for logging with the *tiny* layout.

If you need to customize anything about the server, you'll need to use the Express adapter. I have a script that will let you eject form RAS to Express. https://rmx.fyi/ejectras

Then you'll be able to customize the mogan logger or choose something else entirely.
https://github.com/expressjs/morgan
